src/sentry/integrations/github/client.py

```diff
@@ -573,6 +573,12 @@ def create_comment(self, repo: str, issue_id: str, data: Mapping[str, Any]) -> J
         endpoint = f"/repos/{repo}/issues/{issue_id}/comments"
         return self.post(endpoint, data=data)
 
+    def update_comment(
```

Comment:
i'm not sure how to test this. `create_comment` is called in a different function and that other function is tested, but `create_comment` alone isn't tested